JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -131,14 +130,14 @@ private fun DrawerButton(
     action: () -> Unit
 ) {
     val textIconColor = if (isSelected) {
-        +themeColor { primary }
+        (+MaterialTheme.colors()).primary
```

Comment:
Can all these +MaterialTheme.colors() in this block be extracted into one? I.e:

```
val colors = +MaterialTheme.colors()

if () {
    colors.primary
} else {
    colors.onSurface.copy..
}
```

etc

Same throughout these changes and for typography too